src/components/UserAvatar.js

```diff
@@ -7,15 +7,17 @@ import EStyleSheet from 'react-native-extended-stylesheet'
 
 class UserAvatar extends Component {
   render() {
-
     const guestUserLogo =
       <Image source={require('../../images/simple-avatar.png')} style={styles.avatar} />
 
-    const avatar = ( <Image style={styles.avatar} source={
-        this.props.avatar
-        ? {uri: this.props.avatar}
-        : require('../../images/simple-avatar.png') } /> )
-
+    const avatar = (
+      <Image
+        defaultSource={require('../../images/simple-avatar.png')}
```

Comment:
Why is `defaultSource` needed if there is a ternary to find the same image on `source`?